### DISCUSSION TITLE
What the correct source?

### DISCUSSION THREAD

**Discussion by rrg92** [evildmp/diataxis-documentation-framework#61] [Q&A] [answered]
Hi!

I founds this framework on page https://documentation.divio.com/structure/

But, noted that exists the https://diataxis.fr/

What the correct source that I must reference to give credit?

**Comment by ezio-melotti** [comment, marked as answer]
See https://github.com/evildmp/diataxis-documentation-framework/issues/43#issuecomment-1017745749:
> @evildmp:
> [I created the framework while at Divio](https://diataxis.fr/development/), but have since continued to develop it.

and https://github.com/evildmp/diataxis-documentation-framework/issues/14#issuecomment-1170838014.
> @evildmp:
> [...] I used to work at Divio, which is where Diátaxis started life. https://documentation.divio.com/ has been around for years, and predates https://diataxis.fr/.

---

> What the correct source that I must reference to give credit?

Based on these comments, https://diataxis.fr/ seems the right answer.  You might want to double-check with @evildmp though.

**Reply by rrg92** [reply]
thanks @ezio-melotti .

**Reply by anarcat** [reply]
i was wondering the same thing and actually wondering why the divio source isn't pointing at diataxis.fr now, it seems rather suboptimal to have both sources online right now...
